test/test.getch.ts

```diff
@@ -530,4 +530,37 @@ describe('🍂 defaults & instances', () => {
     scope.done();
     assert.deepStrictEqual(res.data, body);
   });
+
+  it('should allow buffer to be posted', async () => {
+    const pkg = fs.readFileSync('./package.json');
+    const pkgJson = JSON.parse(pkg.toString('utf8'));
+    const scope = nock(url)
+      .matchHeader('content-type', 'application/dicom')
+      .post('/', pkgJson)
```

Comment:
once in transport, the Buffer is properly handled by `node-fetch`, and becomes a string body.

Reply:
sick